packages/create-plugin/templates/github/ci/.github/workflows/ci.yml

```diff
@@ -75,3 +66,31 @@ jobs:
         with:
           version: latest
           args: buildAll
+
+      - name: Check for E2E
+        id: check-for-e2e
+        run: |
+          if [ -d "cypress" ]
+          then
+            echo "has-e2e=true" >> $GITHUB_OUTPUT
+          fi
+
+      - name: Start grafana docker
+        if: steps.check-for-e2e.outputs.has-e2e == 'true'
+        run: docker-compose up -d
+
+      - name: Run e2e tests
+        if: steps.check-for-e2e.outputs.has-e2e == 'true'
```

Comment:
We need to add `id: run-e2e-tests` here if we want to only upload videos for failing tests.